koala-migrate.py

```diff
@@ -8,10 +8,19 @@ def get_koala_connection():
     db_password = os.getenv("KOALA_DB_PASSWORD")
     db_host = os.getenv("KOALA_DB_HOST")
     db_port = os.getenv("KOALA_DB_PORT")
-    return psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+
+    conn =  psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+    return conn
 
 def get_mongoose_connection():
-    return psycopg2.connect(f"postgresql:///undead_mongoose")
+    db_name = os.getenv("DB_NAME")
+    db_user = os.getenv("DB_USER")
+    db_password = os.getenv("DB_PASSWORD")
+    db_host = os.getenv("DB_HOST")
+    db_port = os.getenv("DB_PORT")
+
+    conn = psycopg2.connect(f"dbname={db_name} user={db_user} password={db_password} host={db_host} port={db_port}")
+    return conn
```

Comment:
The database name was changed.
There is a good chance that `return psycopg2.connect(f"postgresql:///mongoose")` may fix the problem.